server/Models/orderItem.js

```diff
@@ -0,0 +1,28 @@
+//ORDER_ITEM : id, pizza_type(pizza_id), quantity, totalAMt
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const { ObjectId } = Schema;
+const DbInstanceConnections = require('../Database/database')
+
+const dbSingleton = new DbInstanceConnections().getInstance();
+const db = mongoose.connection;
+
+db.on('error', console.error.bond(console, 'OrderItem Model connection error'))
+
+db.once('open' , () => {

```

Comment:
I have never needed to do this stuff, is it a type of migration or what is it?

Reply:
The callback is fired once a connection has been successful. I have elaborated more in the comment above

Reply:
Okok!